docs/tokenomics/nns/nns-intro.md

```diff
@@ -1,218 +1,53 @@
-# NNS intro
+# Network Nervous System (NNS)
 
-The NNS is the decentralized autonomous organization (DAO) that governs the Internet Computer Protocol. It is a fully on-chain, decentralized organization and is, for instance, responsible for making protocol level upgrades to continuously improve the Internet Computer Protocol. It does this via an implementation of liquid democracy in which ICP neuron holders vote on proposals that shape the development of the Internet Computer. Once such a proposal is accepted, it is autonomously executed.
+## Overview​
+The [NNS](https://nns.ic0.app/) is the decentralized autonomous organization (DAO) that governs the Internet Computer Protocol. 
```

Comment:
My preference is to not call the Frontend "the NNS" because others from the community might also build a frontend for the NNS. So I would add a sentence, "the on-chain frontend that allows you to interact with it is here" and link to the frontend there

Reply:
That's a good point. I will also note to provide more documentation on how to interact with the governance canister in the future :)